app/src/common/shared/org/mozilla/vrbrowser/browser/engine/SessionUtils.java

```diff
@@ -31,6 +31,8 @@ public static void vrPrefsWorkAround(Context aContext, Bundle aExtras) {
         try (FileOutputStream out = new FileOutputStream(prefFileName)) {
             out.write("pref(\"dom.vr.enabled\", true);\n".getBytes());
             out.write("pref(\"dom.vr.external.enabled\", true);\n".getBytes());
+            // TODO: Unmark `dom.vr.webxr.enabled` when Bug 1614496 is landed.
```

Comment:
Is this comment still valid?

Reply:
Just a test
Just a test

Reply:
We need to uncomment it to enable WebXR

Reply:
But it is uncommented.

Reply:
true, I thought it was `out.write("pref(\"dom.vr.webxr.enabled\", true);\n".getBytes());`. So just remove the comment

Reply:
Yep. I removed it at my last commit `Enable WebXR`